src/sass/components/_about.scss

```diff
@@ -0,0 +1,93 @@
+.about {
+  background-color: $pageBg;
+  background: url('../images/sectionbg1_1x.png') no-repeat;
+  background-size: contain;
+  background-position: center bottom;
+
+  @media (min-width: 768px) {
+    background-image: none;
+  }
+  @media (min-width: 1200px) {
+    background-image: url('../images/sectionbg1_3x.png');
+    background-position-y: 140%;
+  }
+
+  &__tittle-2th {
+    @include font-mixin(Titan One, null, 34, 39px, null, $brand-red);
+    text-align: Center;
+    vertical-align: Top;
+    padding-bottom: 26px;
+
+    @media (min-width: 768px) {
+      font-size: 48;
+    }
+    @media (min-width: 1200px) {
+      font-size: 58;
+    }
+  }
+
+  &__title {
+    @include font-mixin(Titan One, null, 18, 21px, null, $brand-pink);
```

Comment:
font-family musisz dać w pojedynczy cudzysłów: 'test'
font-weight ma 400, a nie null
font-size popraw na 18px
line height obliczasz dzieląc line height/font-size, czyli 21/18, to wychodzi ~1.17 i tak zapisz
letter- spacing nie ma więc wpisujesz "normal"